march_hardware/include/march_hardware/joint.h

```diff
@@ -139,7 +133,7 @@ class Joint
   double absolute_position_ = 0.0;
   double velocity_ = 0.0;
 
-  std::unique_ptr<IMotionCube> imc_ = nullptr;
+  std::unique_ptr<MotorController> controller_ = nullptr;
```

Comment:
What do we do when a motor controller owns multiple joints? I heard the ODrives control 2 joints? I don't see any of that working in the current implementation.

Reply:
We designed the ODrive implementation with this in mind: every Joint is linked to an ODrive object which contains a reference to an ODriveEndpoint object. There is one unique ODrive object for every joint, but only one ODriveEndpoint object for every ODrive. Hence, each ODriveEndpoint object is linked to two ODrive objects. The ODriveEndpoint object handles everything related to communication and the like, similar to the EthercatMaster for our current implementation (except that there are 4 instead of 1).